lib/ruboty/github/actions/merge_pull_request.rb

```diff
@@ -8,12 +8,17 @@ class MergePullRequest < CloseIssue
 
         def close
           request
-          message.reply("Merged #{issue.html_url}")
+          after_merge_message
         end
 
         def request
           client.merge_pull_request(repository, issue_number)
         end
+
+        def after_merge_message
+          message.reply("Merged #{issue.html_url}")
+          message.reply(ENV.fetch('AFTER_MERGE_MESSAGE', nil)) if ENV.fetch('AFTER_MERGE_MESSAGE', nil)
```

Comment:
nits

とはいえ，このままでいい木はする．

```suggestion
          message.reply(ENV.fetch('AFTER_MERGE_MESSAGE', nil)) if ENV.fetch('AFTER_MERGE_MESSAGE', false)
```

Reply:
I fixed it